tests/e2e/rosa/main_test.go

```diff
@@ -0,0 +1,91 @@
+package tests
+
+import (
+	"context"
+	"log"
+
+	"github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/openshift/osde2e-example-test-harness/pkg/metadata"
+	apiextclientset "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+
+	compv1alpha1 "github.com/ComplianceAsCode/compliance-operator/pkg/apis/compliance/v1alpha1"
+)
+
+var _ = ginkgo.Describe("ROSA Compliance Tests", func() {
+	var config *rest.Config
+
+	ginkgo.BeforeEach(func() {
+		var err error
+		config, err = rest.InClusterConfig()
+		Expect(err).NotTo(HaveOccurred(), "unable to load in cluster config")
+
+		// Install the Compliance Operator and wait for it to become available.
```

Comment:
Installing Add-Ons should be done here, right?

This technically runs before both tests, which probably isn't necessary since we're not cleaning up the operator after each spec.